roles/libvirt_manager/tasks/manage_vms.yml

```diff
@@ -81,6 +81,10 @@
       sudo -u zuul mkdir -p /home/zuul/.ssh /home/zuul/src/github.com/openstack-k8s-operators;
       sudo cp ${HOME}/.ssh/authorized_keys /home/zuul/.ssh/;
       chown -R zuul: /home/zuul/.ssh;
+  retries: 5
```

Comment:
fun. OK - care to also update the task "name" from `Configure VMs type {{ vm }}` to something like `Configure VM {{ vm }}` ? Thanks :)

We're being too fast now, aren't we? ;)

Reply:
ack, I'll push and update an get the task name aligned.

Yeah, I think this might be related to the linear. :)
I guess these connections go via the JumpHost? So probably we hit the MaxStartups there ...

Reply:
hmm, `delegate_to` so it should come from the hypervisor, without jumper, as I understand that parameter.